src/main/resources/templates/readme

```diff
@@ -8,10 +8,10 @@ First, we need to implement two sorting algorithms, in this case `MergeSort` and
 
 **You have the following tasks:**
 
-1. ✅[Implement Bubble Sort](testBubbleSort)
+1. &#x2705;[Implement Bubble Sort](testBubbleSort)
```

Comment:
I had to change the unicode symbol to the escaped version.
When saving the unicode symbol to db, it was persistet as random special characters.
This might be an issue with charset/encoding, I felt it was safest to not enforce a db update to solve this issue.